app/upgrades.go

```diff
@@ -23,5 +26,21 @@ func (app *AlloraApp) setupUpgradeHandlers() {
 	for _, handler := range upgradeHandlers {
 		app.UpgradeKeeper.SetUpgradeHandler(handler.UpgradeName,
 			handler.CreateUpgradeHandler(app.ModuleManager, app.Configurator()))
+
+		upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
+		if err != nil {
+			panic(err)
+		}
+
+		if upgradeInfo.Name == v0_7_0.Upgrade.UpgradeName && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+			storeUpgrades := storetypes.StoreUpgrades{
+				Added:   []string{feemarkettypes.ModuleName},
```

Comment:
By adding the `feegrant` module we need to mention it as well, I think here we need to specify the `StoreKey` but it's only a presentation matter as it is the same value, see: https://github.com/skip-mev/feemarket/blob/f1f216e15aa6bcc05075a7079496764820a0f545/x/feemarket/types/keys.go#L7

```suggestion
				Added:   []string{feegrant.StoreKey, feemarkettypes.StoreKey},
```

Reply:
Good catch! I think the fee grant can be a nice addition; it could be an extra feature for topic creators to drive some demand. @kpeluso wdyt?

Reply:
I think it's a great idea, let's do it